xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -827,9 +830,11 @@ Going past this depth throws a <xref:System.Text.Json.JsonException>.
         <ReturnType>System.Text.Json.Serialization.JsonObjectCreationHandling</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Specifies preferred object creation handling for properties when deserializing JSON.
+ When set to <see cref="F:System.Text.Json.Serialization.JsonObjectCreationHandling.Populate" /> all properties which are capable of reusing the existing instance will be populated.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>Only property type is taken into consideration. For example if property is of type <see cref="T:System.Collections.Generic.IEnumerable`1" /> but it is assigned <see cref="T:System.Collections.Generic.List`1" /> it will not be populated because <see cref="T:System.Collections.Generic.IEnumerable`1" /> is not capable of populating.
```

Comment:
```suggestion
        <remarks>Only property type is taken into consideration. For example, if a property is of type <see cref="T:System.Collections.Generic.IEnumerable`1" /> but it is assigned <see cref="T:System.Collections.Generic.List`1" />, it won't be populated because <see cref="T:System.Collections.Generic.IEnumerable`1" /> is not capable of populating.
```